demo/src/static/workers/flight-path-data-decoder.js

```diff
@@ -0,0 +1,55 @@
+importScripts('./util.js');
+const FLUSH_LIMIT = 20000;
+let result = [];
+let index = 0;
+let count = 0;
+
+onmessage = function(e) {
+  const lines = e.data.text.split('\n');
+
+  lines.forEach(function(line) {
+
+    if (!line) {
+      return;
+    }
+
+    var parts = line.split('\t');
+    var coords0 = parts[2].split('\x01').map(function(str) { return decodePolyline(str, 5) });
+    var coords1 = parts[3].split('\x01').map(function(str) { return decodePolyline(str, 1) });
+
+    coords0.forEach(function(lineStr, i) {
+      for (var j = 1; j < lineStr.length; j++) {
+        var prevPt0 = coords0[i][j - 1],
```

Comment:
> We clearly have a linter setup issue with the demo. The root lint script should include the demo folder.

My linter seems to work properly. Let me fix some of the coding style issues for @Pessimistress